metadata.json

```diff
@@ -70,11 +70,15 @@
   ],
   "requirements": [
     {
+      "name": "pe",
+      "version_requirement": ">= 3.2.0 < 5.0.0"
+    },
```

Comment:
This is not needed anymore. the forge evaluates that from the `puppet` requirement.